### PR TITLE
Fix validate failure rule suffix

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by DeanArbel — APPROVED** [review]

File: Packs/AlibabaActionTrail/ReleaseNotes/1_0_15.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#### Correlation Rules
+
+##### Alibaba ActionTrail - multiple unauthorized action attempts detected by a user
+
+Updated the from version to 6.10.0.
```
Comment (ShirleyDenkberg) [inline Packs/AlibabaActionTrail/ReleaseNotes/1_0_15.md:6]:
```suggestion
Updated the fromversion to 6.10.0.
```

File: Packs/AlibabaActionTrail/ReleaseNotes/1_0_15.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+#### XSIAM Dashboards
+
+##### Alibaba Overview Dashboard
+
+Updated the from version to 6.8.0.
```
Comment (ShirleyDenkberg) [inline Packs/AlibabaActionTrail/ReleaseNotes/1_0_15.md:12]:
```suggestion
Updated the fromversion to 6.8.0.
```

File: Packs/CyberArkIdentity/ReleaseNotes/1_0_23.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+#### Correlation Rules
+
+##### CyberArk Failed Logins
+
+Updated the from version to 6.10.0.
```
Comment (ShirleyDenkberg) [inline Packs/CyberArkIdentity/ReleaseNotes/1_0_23.md:6]:
```suggestion
Updated the fromversion to 6.10.0.
```

File: Packs/CyberArkIdentity/ReleaseNotes/1_0_23.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+#### XSIAM Dashboards
+
+##### CyberArk Posture Overview
+
+Updated the from version to 6.8.0.
```
Comment (ShirleyDenkberg) [inline Packs/CyberArkIdentity/ReleaseNotes/1_0_23.md:12]:
```suggestion
Updated the fromversion to 6.8.0.
```

File: Packs/Dropbox/ReleaseNotes/1_0_21.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
+#### Correlation Rules
+
+##### DropBox - Massive File Downloads
+
+Updated the from_version to 6.10.0.
```
Comment (ShirleyDenkberg) [inline Packs/Dropbox/ReleaseNotes/1_0_21.md:6]:
```suggestion
Updated the fromversion to 6.10.0.
```

File: Packs/Dropbox/ReleaseNotes/1_0_21.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+##### DropBox - Massive File Alterations
+
+Updated the from version to 6.10.0.
```
Comment (ShirleyDenkberg) [inline Packs/Dropbox/ReleaseNotes/1_0_21.md:10]:
```suggestion
Updated the fromversion to 6.10.0.
```

File: Packs/Dropbox/ReleaseNotes/1_0_21.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+#### XSIAM Dashboards
+
+##### DropBox Dashboard
+
+Updated the from version to 6.8.0.
```
Comment (ShirleyDenkberg) [inline Packs/Dropbox/ReleaseNotes/1_0_21.md:16]:
```suggestion
Updated the fromversion to 6.8.0.
```

File: Packs/GitLab/ReleaseNotes/2_2_8.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+
+#### Correlation Rules
+
+##### Gitlab - User Permission Changed
+
+Updated the from version to 6.10.0.
```
Comment (ShirleyDenkberg) [inline Packs/GitLab/ReleaseNotes/2_2_8.md:6]:
```suggestion
Updated the fromversion to 6.10.0.
```

File: Packs/GitLab/ReleaseNotes/2_2_8.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+
+#### XSIAM Dashboards
+
+##### Gitlab Overview Dashboard
+
+Updated the from version to 6.8.0.
```
Comment (ShirleyDenkberg) [inline Packs/GitLab/ReleaseNotes/2_2_8.md:12]:
```suggestion
Updated the fromversion to 6.8.0.
```

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_15_25.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+
+#### Correlation Rules
+
+##### Microsoft Defender for Endpoint - Malware Detected
+
+Updated the from version to 6.10.0.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_15_25.md:6]:
```suggestion
Updated the fromversion to 6.10.0.
```

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_15_25.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+
+#### XSIAM Dashboards
+
+##### Microsoft Defender for Endpoint Overview Dashboard
+
+Updated the from version to 6.8.0.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_15_25.md:12]:
```suggestion
Updated the fromversion to 6.8.0.
```

File: Packs/MicrosoftWindowsEvents/ReleaseNotes/1_0_2.md [hunk 12]
```diff
@@ -0,0 +1,6 @@
+
+#### XSIAM Dashboards
+
+##### Microsoft Windows Events
+
+Updated the from version to 6.8.0.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/ReleaseNotes/1_0_2.md:6]:
```suggestion
Updated the fromversion to 6.8.0.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @ilappe @bziser Doc review completed.

**Review by bziser — APPROVED** [review]

**Pull request merged**